### PR TITLE
Update libevent from 2.1.8 to 2.1.11

### PR DESCRIPTION
**Pull request by cstrouse**
This version of libevent contains ABI-breaking changes. Tested on ARM.

### CODE REVIEW

**Comment by uberhacker** [comment]
We need to test this update against the memcached, qtbase, tmux and transmission package binaries.

File: packages/libevent.rb [hunk 1]
```diff
@@ -1,33 +1,24 @@
...
 
 class Libevent < Package
-  description 'The libevent API provides a mechanism to execute a callback function when a specific event occurs on a file descriptor or after a timeout has been reached.'
+  description 'Event notification library'
   homepage 'http://libevent.org/'
-  version '2.1.8-0'
-  source_url 'https://github.com/libevent/libevent/releases/download/release-2.1.8-stable/libevent-2.1.8-stable.tar.gz'
-  source_sha256 '965cc5a8bb46ce4199a47e9b2c9e1cae3b137e8356ffdad6d94d3b9069b71dc2'
+  version '2.1.11'
+  source_url 'https://github.com/libevent/libevent/archive/release-2.1.11-stable.tar.gz'
```
Comment (uberhacker) [inline packages/libevent.rb]: You probably don't need `./autogen.sh` with the release download.

Comment (cstrouse) [reply packages/libevent.rb]: When I do updates I always download and examine the release tarball. This one didn't have configure present so autogen.sh was needed.

Comment (uberhacker) [reply packages/libevent.rb]: Did you try `https://github.com/libevent/libevent/releases/download/release-2.1.11-stable/libevent-2.1.11-stable.tar.gz`?

**Pull request closed without merging**